examples/mnist/notebook.jl

```diff
@@ -63,8 +65,6 @@ struct MyConvBuilder
     channels3::Int
 end
 
-make2d(x::AbstractArray) = reshape(x, :, size(x)[end])
```

Comment:
There is another instance of this below around line 173, should it be also replaced with `MLUtils.flatten`?

Reply:
Okay, it looks like `make2d` is not used, and we can use `vec` for `make1d`. So I've done that.